src/player_activity.cpp

```diff
@@ -104,8 +105,20 @@ cata::optional<std::string> player_activity::get_progress_message( const avatar
             type == activity_id( "ACT_CHOP_LOGS" ) ||
             type == activity_id( "ACT_CHOP_PLANKS" )
           ) {
+            const int percentage = ( ( moves_total - moves_left ) * 100 ) / moves_total;
+
             extra_info = string_format( "%d%%", percentage );
         }
+
+        if( type == activity_id( "ACT_BUILD" ) ) {
+            partial_con *pc = g->m.partial_con_at( g->m.getlocal( u.activity.placement ) );
+            if( pc ) {
+                int counter = std::min( pc->counter, 10000000 );
```

Comment:
I wonder why the next variable is const, bu this one isn't?

Reply:
The original percentage line above was const.  I just copied that line and modified it.

I'm not a huge fan of const'ing everything under the sun so don't typically add it to variables I declare.